community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -102,6 +102,10 @@
             "allows an upgrade to be performed.")
     public static final Setting<Boolean> allow_store_upgrade = setting("dbms.allow_format_migration", BOOLEAN, FALSE );
 
+
+    @Description( "Specifies custom storage format name. If not set default format will be used." )
```

Comment:
"custom" seems weird here. Also the "name" is probably unnecessary, isn't it? How about "The storage format."?